p2p/server.go

```diff
@@ -728,39 +694,74 @@ func (srv *Server) run(dialstate dialer) {
 		}
 	}
 
-	isValNode := func(ID enode.ID) bool {
-		if _, ok := valNodes[ID]; ok {
-			return true
+	addStatic := func(n *enode.Node, label string) {
+		if _, ok := static[n.ID()]; !ok {
+			static[n.ID()] = make(map[string]bool)
+		}
+		static[n.ID()][label] = true
+
+		// If already connected, updated val peer counters and set the validatorConn flag in the connection
```

Comment:
How is the flag getting set? Or are you referencing the labels?

Reply:
Good catch!  This is an old comment for the original validator connections.